home/models.py

```diff
@@ -212,19 +224,25 @@ class Meta:
 
     def clean(self, value):
         result = super().clean(value)
+        num_vars_in_msg = result["message"].count("{{")
```

Comment:
We could maybe use regex here to do a bit of a better match. ie. `len(re.findall("{{\d+}}", result["message"]))`

Reply:
Cool, thanks